### PR TITLE
Paul/multi input runs

### PR DESCRIPTION
**Pull request by paulcjh**

### CODE REVIEW

**Comment by paulcjh** [comment]
Addition of GPT J showing multi inputs
Multi inputs
Non str input types for runs

File: pipeline/api/cloud.py [hunk 1]
```diff
@@ -57,6 +58,11 @@ def upload_file(self, file_or_path, remote_path) -> FileGet:
         else:
             return self._post_file("/v2/files/", file_or_path, remote_path)
 
+    def upload_data(self, file_or_path, remote_path) -> DataGet:
+        uploaded_file = self.upload_file(file_or_path, remote_path)
+        uploaded_data = self._post("/v2/data/", uploaded_file.dict())
```
Comment (alexpearce) [inline pipeline/api/cloud.py:63]: Could you remove the trailing slash here? Most top routes do not have one (the exception being `POST /v2/files/`, which we should fix).

**Review by alexpearce — APPROVED** [review]
Just a URL change please.

File: pipeline/api/cloud.py (hunk 1 above)
Comment (paulcjh) [reply pipeline/api/cloud.py:63]: Sure, and will change on the top node too (I added a "/" to match files)

**Review by alexpearce — APPROVED** [review]

**Pull request merged**